src/main/java/frc/robot/commands/AutoAlignCommands/SwitchLimelightStream.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
maybe keep? if we want to keep USB camera as the "main" one then we don't need it